docs/src/use/migrate-to-9.0.0.md

```diff
@@ -306,6 +306,15 @@ ESLint v9.0.0 removes multiple deprecated methods from the `context` object and
 |`context.getTokensBefore()`|`sourceCode.getTokensBefore()`|
 |`context.getTokensBetween()`|`sourceCode.getTokensBetween()`|
 |`context.parserServices`|`sourceCode.parserServices`|
+|`context.getDeclaredVariables()`|`sourceCode.getDeclaredVariables()`|
+
+In addition to the methods in the above table, there are several other methods that are also moved but required different method signatures:
+
+|**Removed on `context`**|**Replacement(s) on `SourceCode`**|
+|-----------------------|--------------------------|
+|`context.getAncestors()`|`sourceCode.getAncestors()`|
+|`context.getScope()`|`sourceCode.getScope()`|
+|`context.markVariableAsUsed()`|`sourceCode.markVariableAsUsed()`|
```

Comment:
Maybe we can mention the signature as well, WDYT?
```suggestion
|`context.getAncestors()`|`sourceCode.getAncestors(node)`|
|`context.getScope()`|`sourceCode.getScope(node)`|
|`context.markVariableAsUsed(name)`|`sourceCode.markVariableAsUsed(name , node)`|
```

Reply:
That was my first idea too, but then I thought it might look inconsistent with the first table, which doesn't show signatures, so I'm not sure.

Reply:
I believe the inconsistency should be fine as in the first table, signatures haven't changed. Would like to know other opinions.

Reply:
Given that we specifically mention these methods have different signatures, I think it makes sense to actually show the signature. :+1:

Reply:
Okay, I'll apply the suggestion now 👍